docs/cookbook/ignore-occurs-requirements.md

```diff
@@ -0,0 +1,57 @@
+---
+title:  Optionally ignoring the occurs requirement for fields
+---
+# {{ page.title }}
+_(Applies to Ion Schema 1.0.)_
+
+Suppose you have a type definition for a struct, and sometimes you want to validate the fields without enforcing the
```

Comment:
Just wanted to understand the use case here,
Is this used if you want to validate both ways with the occurs requirement and without the occurs requirement right? Because otherwise we could just set occurs with optional/required.

Reply:
Yes, that's right. This is for if you want to be able to validate both with and without the occurs requirements.

Reply:
From what I understand - yes. When we are trying to validate fields with expected types (no enforcement of the required-ness of any specific fields), we can validate against the first example.